service/storage_proxy.cc

```diff
@@ -4864,7 +4866,7 @@ future<bool> storage_proxy::cas(schema_ptr schema, shared_ptr<cas_request> reque
                 ++get_stats().cas_failed_read_round_optimization;
 
                 auto pr = partition_ranges; // cannot move original because it can be reused during retry
-                auto cqr = co_await query(schema, cmd, std::move(pr), cl, query_options);
+                auto cqr = co_await query(schema, cmd, std::move(pr), cl, db::allow_per_partition_rate_limit::no, query_options);
```

Comment:
Maybe we should move it to read_cmd? That's already an extension friendly structure and can eliminate lots of boilerplate (and runtime cost of passing parameters).

Reply:
I put `allow_limit` into `read_command`... However, it's a coordinator-only parameter (it is converted to `db::per_partition_rate_limit::info` during coordinator logic), so it it resulted in only slight reduction of boilerplate (it's not serialized in rpc, by the way).